package.json

```diff
@@ -25,16 +25,15 @@
   "scripts": {
     "build": "npm run setup && npm run preprocess",
     "build-test": "npm run setup && npm run preprocess-test",
-    "lint": "suitcss -c test/config.json index.css build/lint.css && rm build/lint.css",
+    "lint": "suitcss index.css build/lint.css -e && rm build/lint.css",
```

Comment:
I switched it around so the flag is only used for the CI - https://github.com/suitcss/components-button/pull/22/commits/bbdf87db39725b6ff8af796f60502c2794294137